ada_2_fhir-r4/zibs2020/fhir_instance/nl-core-CareAgreement-01-Condition-01.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Condition xmlns="http://hl7.org/fhir">
+   <id value="nl-core-CareAgreement-01-Condition-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-Problem"/>
+   </meta>
+   <clinicalStatus>
+      <coding>
+         <system value="http://terminology.hl7.org/CodeSystem/condition-clinical"/>
+         <code value="active"/>
+         <display value="Active"/>
+      </coding>
+   </clinicalStatus>
+   <verificationStatus>
+      <coding>
+         <system value="http://terminology.hl7.org/CodeSystem/condition-ver-status"/>
+         <code value="confirmed"/>
+         <display value="Confirmed"/>
+      </coding>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="410605003"/>
+         <display value="Aanwezigheid bevestigd"/>
+      </coding>
+   </verificationStatus>
```

Comment:
The QA tool gives the following error:

```
-  error at Condition.verificationStatus (14, 24):
     Condition.verificationStatus.coding:verificationStatusCodelist: minimum required = 1, but only found 0 (from http://nictiz.nl/fhir/StructureDefinition/nl-core-Problem)
```

The code is included in the example; perhaps a QA error?